### ISSUE TITLE
Occlusion Culling Plane component is green and prints a warning to the Console when added or modified

### ISSUE THREAD

**Issue by LB-ArturZieba** [o3de/o3de#7815]
**Describe the bug**
Occlusion Culling Plane component is green instead of black and prints a warning to the Console whenever it is added to an entity, or it's parameters are updated. It should be noted that this does not occur on Linux.

Printed warning message:
`[Warning] (MaterialAsset) - This material is based on version '1' of ['materials/types/standardpbr.azmaterialtype' AssetId={15F89219-4AEA-52A0-A520-A818249665CA}:0 Type={CD7803AB-9C4C-4A33-9A14-7412F1665464}], but the material type is now at version '4'. Automatic updates are available. Consider updating the .material source file for '{3F28138D-27D7-5B62-977E-F49C08B77B7D}:0'.`

**Steps to reproduce**
1. Create an entity and add an Occlusion Culling Plane component to it.

**Expected behavior**
Occlusion Culling Plane is black.

**Actual behavior**
Occlusion Culling Plane is green and prints an error to the Console when added.

**Video**

https://user-images.githubusercontent.com/86952082/155114199-5ecd609d-c1ab-4bd7-8576-039b4824a423.mp4

**Found in Branch**
Development (90d5f57)

**Desktop**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU AMD Ryzen 5 3600
 - GPU NVIDIA GeForce RTX 2060 SUPER
 - Memory 16GB

**Comment by santorac** [comment]
I can fix the warning, but @LB-ArturZieba can you clarify why you think the visualization should be black? The material being used is explicitly set to be green, it appears to be as-designed.

**Comment by LB-ArturZieba** [comment]
Sorry my mistake, the first time I tested the Occlusion Culling Plane was on Linux and it was black so when I went back to Windows I assumed that green color is a bug. I did not realize back then that it is a separate material (the black material issue is reported here: https://github.com/o3de/o3de/issues/7519). The issue can be closed when the warning print is fixed, I've confirmed that the OcclusionCullingPlaneVisualization.material is green on the Development (48e3b2a) build.
